src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,34 +1,71 @@
-export const TodoFilter = () => (
-  <form className="field has-addons">
-    <p className="control">
-      <span className="select">
-        <select data-cy="statusSelect">
-          <option value="all">All</option>
-          <option value="active">Active</option>
-          <option value="completed">Completed</option>
-        </select>
-      </span>
-    </p>
-
-    <p className="control is-expanded has-icons-left has-icons-right">
-      <input
-        data-cy="searchInput"
-        type="text"
-        className="input"
-        placeholder="Search..."
-      />
-      <span className="icon is-left">
-        <i className="fas fa-magnifying-glass" />
-      </span>
-
-      <span className="icon is-right" style={{ pointerEvents: 'all' }}>
-        {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-        <button
-          data-cy="clearSearchButton"
-          type="button"
-          className="delete"
+import React from 'react';
+import { SelectorOption } from '../../enum';
+
+type Props = {
+  setCurrentSelector: (value: string) => void;
+  setSearchQuery: (value: string) => void;
+  searchQuery: string;
+};
+
+export const TodoFilter: React.FC<Props> = ({
+  setCurrentSelector,
+  setSearchQuery,
+  searchQuery,
+}) => {
+  const handlerSelector = (event: React.ChangeEvent<HTMLSelectElement>) => {
+    setCurrentSelector(event.target.value);
+  };
+
+  const handlerSearchChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    const { value } = event.target;
+
+    setSearchQuery(value);
+  };
```

Comment:
Why do you do destructuring for the event of the first handler and not for the second?)
In this case, it can not be done, because the value value is used only once in each handler

Reply:
Cause ESLint throw error with: "Make destructuring for that event", i will change that in patch.